examples/messenger-batch/README.md

```diff
@@ -0,0 +1,46 @@
+# Messenger batch
+
+## Install and Run
+
+Download this example or clone [bottender](https://github.com/Yoctol/bottender).
+
+```
+curl https://codeload.github.com/Yoctol/bottender/tar.gz/master | tar -xz --strip=2 bottender-master/examples/messenger-batch
+cd messenger-batch
+```
+
+Install dependencies:
+
+```
+npm install
+```
+
+You must put `accessToken`, `appSecret` and `verifyToken` into `bottender.config.js`.
+
+After that, you can run the bot with this npm script:
+
+```
+npm run dev
+```
+
+This command will start server for bot developing at `http://localhost:5000`.
+
+## Set webhook
+
+While the server running, you can run following command with global `bottender` to set up the webhook:
+
+```
+bottender messenger webhook set -w <YOUR_WEBHOOK_URL>
+```
+
+If you want to expose the server on your local development machine and get a secure URL, [ngrok](https://ngrok.com/) or [localtunnel](https://localtunnel.github.io/www/) may be good tools for you.
+
+> Note: You must put `appId`, `appSecret` and `verifyToken` into `bottender.config.js` before running this command.
+
+## Idea of this example
+
+This example is a bot which sents multiple requsets (up to 50) in one batch using `batchConfig`.
```

Comment:
We should add some docs about `batchConfig` settings and a link to `messenger-batch` repo.